### ISSUE TITLE
Can't open project that I have created

### ISSUE THREAD

**Issue by Museumdatacleaner** [OpenRefine/OpenRefine#1369]
I can't open a project that I have just created, whatever the size of the file. It loads through to a window saying it is loading, but never loads the data.

**Comment by thadguidry** [comment]
@Museumdatacleaner Can you provide a screenshot ?
Are you importing a previous OpenRefine project (a .tar.gz file) ?
Any errors you see in the Console or Terminal that OpenRefine is running in ?

**Comment by Museumdatacleaner** [comment]
Hey, 
I am not importing a new project (have just started using openrefine on this machine). Below are example projects.
![openrefine](https://user-images.githubusercontent.com/34109164/33401223-70a3aa4c-d5bd-11e7-8e3d-b26280d95ec6.JPG)
When I try to open them it sits on this screen.
![openrefinenot loading](https://user-images.githubusercontent.com/34109164/33401469-41b18d02-d5be-11e7-8358-85f3d252f832.JPG)
![console](https://user-images.githubusercontent.com/34109164/33401489-567d83b2-d5be-11e7-9317-c4365b935b6b.JPG)
No errors that I can see.

**Comment by thadguidry** [comment]
Looks like your using IE, some older version...I'm wondering if their are critical Javascript errors in its console ?

https://msdn.microsoft.com/en-us/library/dd565630(v=vs.85).aspx

Can you also try newest Firefox or Chrome latest ?

**Comment by Museumdatacleaner** [comment]
Success! Must be IE browser, as it opens on Chrome. I should stop using IE in general...
Thanks!

**Comment by jackyq2015** [comment]
Poor IE. :)

**Comment by thadguidry** [comment]
@Museumdatacleaner Edge is pretty awesome actually on Windows 10 and OpenRefine works fine on it.

**Comment by AnnVanCamp** [comment]
I can't open the projects I've made either (not with Edge nor with Chrome) although they are still stored on my local disk.
![image](https://user-images.githubusercontent.com/75479572/102218027-b0cf4400-3edd-11eb-9bee-16923f735d08.png)
When I click on "Browse workspace directory" I get this. Which file in the project do I have to select to open the project in the webbrowser?
![image](https://user-images.githubusercontent.com/75479572/102218197-e8d68700-3edd-11eb-9506-37d6d8bfd5d5.png)
I'm sorry if this question is really stupid, but I've just started to work with OpenRefine.
